### ISSUE TITLE
[Imprv] add functionality "created" as a state to module ntnx_vms

### ISSUE THREAD

**Issue by rafiwicht** [nutanix/nutanix.ansible#158]
Hello everyone

It would be nice to have the possibility to only create a VM and not start it. I have a use case, where I have to create the DHCP reservation before starting the VM.

If you create a VM with present, it will always be started.

Either have an additional property (for ex. called power_state) or extend the existing state property.

Best regards

Rafi

**Comment by premkarat** [comment]
Hi @rafiwicht this is coming in future release. Please stay tuned. We will extend the state property

**Comment by premkarat** [comment]
implemented
